source/utilities/realtime/dataman/DumpMan.cpp

```diff
@@ -0,0 +1,83 @@
+#include <iostream>
+
+#include "DumpMan.h"
+
+std::shared_ptr<adios::realtime::DataManBase> getMan()
+{
+    return std::shared_ptr<adios::realtime::DataManBase>(
+        new adios::realtime::DumpMan);
+}
+
+namespace adios
+{
+namespace realtime
+{
+
+int DumpMan::init(json p_jmsg)
+{
+    if (p_jmsg["dumping"] != NULL)
+    {
+        m_dumping = p_jmsg["dumping"].get<bool>();
```

Comment:
It would be good to keep in the sense that I can use it as an example to add something else later as a dynamic library. Plus it is a quite trivial implementation and won't cause problems.